redact_sample.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import csv
+import logging
+import os
+import sys
+from typing import Iterable
+
+from generate_sample_file import SampleGenerator
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+SampleGen = SampleGenerator()
+SampleGen.ESTAB_TYPES = ['HALL OF RESIDENCE', 'CARE HOME', 'HOSPITAL', 'HOSPICE', 'MENTAL HEALTH HOSPITAL',
+                         'MEDICAL CARE OTHER', 'BOARDING SCHOOL', 'HOTEL',
+                         'YOUTH HOSTEL', 'HOSTEL', 'EDUCATION OTHER', 'PRISON', 'STAFF ACCOMMODATION', 'CAMPHILL',
+                         'HOLIDAY PARK', 'HOUSEHOLD', 'RESIDENTIAL CARAVAN', 'RESIDENTIAL BOAT', 'GATED APARTMENTS',
+                         'FOREIGN OFFICES', 'CASTLES', 'EMBASSY', 'CARAVAN', 'MARINA']
+SampleGen.read_words()
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Load a sample file into response management.')
```

Comment:
Just need to tweak the description here

Reply:
Tweak = change it completely?

Reply:
Now its: description='Redact a sample file.'